packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -2,12 +2,11 @@ import { LogTypes, StorageTypes } from '@requestnetwork/types';
 import Utils from '@requestnetwork/utils';
```

Comment:
all IPFS logic in this file has been moved to ipfs-storage

Reply:
Clever, thanks